neonvm/controllers/virtualmachine_controller.go

```diff
@@ -1400,6 +1400,16 @@ func podSpec(virtualmachine *vmv1.VirtualMachine, sshSecret *corev1.Secret, conf
 				containerMgr := corev1.Container{
 					Image: image,
 					Name:  "neonvm-container-mgr",
+					SecurityContext: &corev1.SecurityContext{
+						Privileged: &[]bool{true}[0],
```

Comment:
Yes, I wasn't able to get any combination of capabilities to work without privileged (see https://github.com/kubernetes/kubernetes/issues/121190)

Reply:
Huh, we already had rw cgroups access without privileged before, if that's the problem. (we manually mounted the entire `/sys/fs/cgroup`).

I have some ideas here, will try that out today.

Reply:
Tried it out. Didn't work - now realizing eBPF access requires you to be acting as root on the host (i.e. privileged), because otherwise it's too easy to break out of the container.

Weird that CAP_BPF doesn't work though...

If we set `Privileged: true`, the other capabilities are not needed.

Reply:
Got rid of the capabilities